tests/unit/test_store.py

```diff
@@ -74,40 +85,39 @@ def test_store_can_create_s3_fsspec_session(self):
                 responses.add(
                     responses.GET,
                     daac["s3-credentials"],
-                    json={
-                        "accessKeyId": "sure",
-                        "secretAccessKey": "correct",
-                        "sessionToken": "whynot",
-                    },
+                    json=mock_creds,
                     status=200,
                 )
         responses.add(
             responses.GET,
             "https://urs.earthdata.nasa.gov/profile",
-            json={},
+            json=mock_creds,
             status=200,
         )
 
         store = Store(self.auth)
         self.assertTrue(isinstance(store.auth, Auth))
         for daac in ["NSIDC", "PODAAC", "LPDAAC", "ORNLDAAC", "GES_DISC", "ASF"]:
             s3_fs = store.get_s3fs_session(daac=daac)
-            self.assertEqual(type(s3_fs), type(fsspec.filesystem("s3")))
+            assert isinstance(s3_fs, s3fs.S3FileSystem)
+            assert s3_fs.storage_options == expected_storage_options
```

Comment:
Nice! Do we still need to inherit from `unittest.TestCast` now that we're using assert statements?

Reply:
We're still using `unittest` style code elsewhere, for example `self.assertTrue(isinstance(store.auth, Auth))` a few line above